src/Microsoft.AspNet.Http.Extensions/ResponseExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Http.Features;
+
+namespace Microsoft.AspNet.Http.Extensions
+{
+    public static class ResponseExtensions
+    {
+        public static void Clear(this HttpResponse response)
+        {
+            response.StatusCode = 200;
+            response.HttpContext.Features.Get<IHttpResponseFeature>().ReasonPhrase = null;
```

Comment:
No, we consider reason phrase obsolete/useless. 99% of users just set it to the default value for the given status code. It has also been removed in HTTP/2.